frontend/src/components/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+
+import { DayHeaders } from './DayHeaders'
+import { MonthlyContainer } from './MonthlyContainer'
+
+const CalenderContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+`
+
+const YearContainer = styled.div`
+  width: calc(47.14px * 7);
+  font-weight: 600;
+  color: #58b994;
+  text-align: center;
+  padding-top: 10px;
+`
+
+const MonthContainer = styled.div`
+  padding-bottom: 5px;
+`
+
+const MonthsHeaderContainer = styled.div`
+  width: 93px;
+  color: #58b994;
+  margin-top: 10px;
+  margin-left: 10px;
+  padding-bottom: 20px;
+  font-weight: 600;
+`
+
+const DatesGridContainer = styled.div`
+  display: grid;
+  grid-template-columns: 47.14px 47.14px 47.14px 47.14px 47.14px 47.14px 47.14px;
+  grid-template-rows: 40px 40px 40px 40px 40px 40px;
+`
+
+// this component takes in an array of events or an array of dates that has events
+export const Calendar = () => {
+  // event dates are stored as UNIX format
+  // dummy dates with events : 5th Feb, 5th Mar, 29th Mar, 2nd Apr
+  const eventDays: number[] = [1644072529, 1646491729, 1648565329, 1648910929]
+
+  // dummy dates with events : 1st Mar, 3rd Mar, 31st Mar, 31st Jan (In component friendly format)
+  const processedDates: number[] = [204, 301, 303, 331, 131]
+
+  // conert unix dates into component friendly format
+  const convertDates = (unprocessedDate: number) => {
+    const month = new Date(unprocessedDate * 1000).getMonth() + 1
+    const day = new Date(unprocessedDate * 1000).getDate()
+    const processedDate = month * 100 + day
+    processedDates.push(processedDate)
+    return processedDate
+  }
+
+  eventDays.forEach((date) => convertDates(date))
+
+  const today = new Date()
+  const currentYear = today.getFullYear()
+  const Jan = new Date('January 1, 2000 01:00:00').toDateString().slice(4, -7)
```

Comment:
should the date be hardcoded? all januarys equate to this?

Reply:
Resolved in 5d138ac